main/coursecopy/recycle_course.php

```diff
@@ -55,14 +59,42 @@
         $course = $cb->build();
     }
     $recycle_type = '';
+    $fullDelete = 0;
+    $courseCodeConfirmation ='';
+    if (isset($_POST['course_code_confirmation']))
+        {
+            $courseCodeConfirmation = $_POST['course_code_confirmation'];
+        }
```

Comment:
Codacy found an issue: [Line indented incorrectly; expected 4 spaces, found 8](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5721344)

Reply:
Spacing needs to be corrected